src/components/MessagesList/MessagesGroup/Message/MessagePart/PollCard.vue

```diff
@@ -39,8 +39,30 @@
 		class="poll-card"
 		role="button"
 		@click="openPoll">
+		<div v-if="showPollStatus" class="poll-card__status">
+			<NcChip
+				:variant="pollImportance"
+				no-close
+				size="small">
+				<template #icon>
+					<IconCircle
+						:size="10" />
+				</template>
+				{{ poll?.status === POLL.STATUS.OPEN ? t('spreed', 'Open poll') : t('spreed', 'Closed poll') }}
+			</NcChip>
+			<NcChip
+				v-if="poll?.resultMode === POLL.MODE.HIDDEN"
+				variant="tertiary"
+				no-close
+				size="small">
+				<template #icon>
+					<IconAccountCircleOutline :size="15" />
+				</template>
+				{{ t('spreed', 'Anonymous') }}
```

Comment:
I'd use existing string, otherwise add a translators hint
```suggestion
				{{ t('spreed', 'Anonymous poll') }}
```

Reply:
I was just afraid that the word poll will be too redundant